jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/pojos/tim/DeserializedRawTim.java

```diff
@@ -0,0 +1,39 @@
+package us.dot.its.jpo.geojsonconverter.pojos.tim;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Generated;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import java.util.List;
+
+import us.dot.its.jpo.geojsonconverter.pojos.ProcessedValidationMessage;
+import us.dot.its.jpo.ode.model.OdeMessageFrameData;
+
+/**
+ * Represents a deserialized raw TIM message with validation results.
+ * <p>
+ * odeTimMessageFrameData - The deserialized ODE TIM message frame data
+ * <p>
+ * validatorResults - The validation results from JSON schema validation
+ * <p>
+ * validationFailure - Whether validation failed
+ * <p>
+ * failedMessage - The failure message if validation failed
+ */
+@Data
+@Generated
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Slf4j
+public class DeserializedRawTim {
```

Comment:
question: would this be easier to just extend the new J2735 pojo and add the validation messages to it? Or is there a reason to not use those pojos for this instance?

Reply:
It still uses the "new" j2735 POJO by using the ODE defined "OdeMessageFrameData" which includes asn-pojo messageframe references. This wrapper class will be used in the future to hold validation errors and any failure messages.

Reply:
I'll add that all of the "DeserializedRawX" classes are composite objects that are used to hold intermediate values during processing within the topology with types that are more informative than something like Tuple<OdeMessageFrameData, List<>, Boolean, String>, but that aren't part of the public outputs of the app. Essentially it's used as a named tuple, so I think it makes more sense to use composition and not inheritance for it.